.babelrc

```diff
@@ -2,15 +2,21 @@
 	"presets": [
 		"@babel/env",
 		"@babel/typescript",
```

Comment:
Isn't it named @babel/preset-typescript?

Reply:
Renamed for consistency